commands/apps_dev_config.go

```diff
@@ -161,6 +156,29 @@ func outputValidAppDevKeys() string {
 	return strings.Join(keys, ", ")
 }
 
+type appDevConfig struct {
+	contextDir string
+	cmdConfig  *CmdConfig
+	viper      *viper.Viper
+}
+
+func (c *appDevConfig) CacheDir(component string) string {
+	return filepath.Join(c.contextDir, ".do", "cache", component)
+}
+
+func (c *appDevConfig) EnsureCacheDir(ctx context.Context, component string) error {
+	err := os.MkdirAll(filepath.Join(c.contextDir, ".do", "cache", component), os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	return ensureStringInFile(filepath.Join(c.contextDir, ".do", "cache", ".gitignore"), "./...")
```

Comment:
does this work in `.gitignore`? what about:

```suggestion
	return ensureStringInFile(filepath.Join(c.contextDir, ".do", ".gitignore"), "/cache")
```